plugins/suspendmanager.cpp

```diff
@@ -522,8 +542,9 @@ class SuspendManager {
 
             df::building* exit = nullptr;
             for (auto npos : neighbors | transform(around(pos))) {
-                if (!isSuitableAccess(npos))
-                    continue; // non walkable neighbour, not an exit
+                if (!isPotentialSuitableAccess(npos))
+                    // non walkable neighbour, nor planned to become one, not an exit
+                    continue;
```

Comment:
If I understand correctly, then this change allows filling multi-z dead ends. However, if the wall built on the level below isn't actually accessible, this may actually cause dead ends to be closed off. But if this allows designating multi-z width-1 walls from one adjacent stairwell for "scaffolding", I'm all for it.

Reply:
It's intended to adress multi-z dead ends yes (it works better when there is no cat):
![multi-level-deadend](https://github.com/DFHack/dfhack/assets/630159/377938dc-415c-4b36-b27f-26b51a0a0810)

> However, if the wall built on the level below isn't actually accessible, this may actually cause dead ends to be closed off

I think if a wall below is inaccessible, this blocks the whole process until the player have restored access to it, which feels okayish

Reply:
"Dead end" doesn't require a wall. So this should also work for free-standing multi-z walls (i.e. to close off part off a cavern), which would be a great plus, certainly making up for some minor inconveniences in rare cases. :clap: